packages/nimble-components/src/toggle-button/tests/toggle-button-matrix.stories.ts

```diff
@@ -47,32 +54,60 @@ type CheckedState = (typeof checkedStates)[number];
 
 // prettier-ignore
 const component = (
+    [interactionName, interaction]: InteractionState,
     [iconVisible, labelVisible, endIconVisible]: PartVisibilityState,
     [checkedName, checked]: CheckedState,
     [disabledName, disabled]: DisabledState,
     [appearanceName, appearance]: AppearanceState
 ): ViewTemplate => html`
     <${toggleButtonTag}
+        class="${() => interaction}"
         appearance="${() => appearance}"
         ?disabled=${() => disabled}
         ?content-hidden=${() => !labelVisible}
         ?checked=${() => checked}
         style="margin-right: 8px; margin-bottom: 8px;">
             ${when(() => iconVisible, html`<${iconKeyTag} slot="start"></${iconKeyTag}>`)}
-            ${() => `${checkedName} ${appearanceName} Toggle Button ${disabledName}`}
+            ${() => `${interactionName} ${checkedName} ${appearanceName} Toggle Button ${disabledName}`}
             ${when(() => endIconVisible, html`<${iconArrowExpanderDownTag} slot="end"></${iconArrowExpanderDownTag}>`)}
     </${toggleButtonTag}>
 `;
 
 export const toggleButtonThemeMatrix: StoryFn = createMatrixThemeStory(
     createMatrix(component, [
+        nonInteractionStates,
         partVisibilityStates,
         checkedStates,
         disabledStates,
         appearanceStates
     ])
 );
 
+export const toggleButtonInteractionsThemeMatrix: StoryFn = createMatrixThemeStory(
+    createMatrix(
+        component,
+        [
+            interactionStates,
+            [[false, true, false]],
+            checkedStates,
+            disabledStates,
+            appearanceStates
+        ],
+        // Only interaction relevant to disabled controls is hover
+        (
+            interactionState: InteractionState,
+            _partVisibilityState: PartVisibilityState,
+            _checkedState: CheckedState,
+            disabledState: DisabledState
+        ) => {
+            return disabledInteractionsFilter(
```

Comment:
Can this use `disabledInteractionsFilter` directly if the states were reordered?

Reply:
Yes, I'll reorder them.